packages/jellyfish-transaction-signature/src/index.ts

```diff
@@ -1,21 +1,20 @@
 import { EllipticPair, dSHA256, HASH160 } from '@defichain/jellyfish-crypto'
+import {
+  Script,
+  Transaction,
+  TransactionSegWit,
+  Vin,
+  Vout,
+  Witness,
+  OP_CODES,
+  OP_PUSHDATA,
+  CWitnessProgram,
+  WitnessProgram,
+  DeFiTransactionConstants,
+  SIGHASH,
+  CVoutV4
+} from '@defichain/jellyfish-transaction'
 import { SmartBuffer } from 'smart-buffer'
-import { Script, Transaction, TransactionSegWit, Vin, Vout, Witness } from './tx'
-import { OP_CODES, OP_PUSHDATA } from './script'
-import { CWitnessProgram, WitnessProgram } from './tx_segwit'
-import { DeFiTransactionConstants } from './index'
-import { writeVarUInt } from './buffer/buffer_varuint'
-import { ONE_HUNDRED_MILLION, writeBigNumberUInt64 } from './buffer/buffer_bignumber'
-
-export enum SIGHASH {
```

Comment:
Considered part of TransactionSegWit definition, moved to `jellyfish-transaction`/`tx_segwit`